frontend/src/scenes/notebooks/NotebookPanel/NotebookPanel.scss

```diff
@@ -0,0 +1,60 @@
+@import '../../../styles/mixins';
+
+.NotebookPanel {
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+    background-color: var(--bg-3000);
+    overflow: hidden;
+
+    .NotebookPanelDropzone {
+        border: 2px dashed var(--border-3000);
+        border-radius: var(--radius);
+        margin: 0.5rem;
+        transition: all 150ms;
+        height: 4rem;
+        display: flex;
+
+        .NotebookPanelDropzone__message {
+            flex: 1;
+            display: flex;
+            flex-direction: column;
+            justify-content: center;
+            align-items: center;
+            font-weight: 700;
+            font-size: 1rem;
+            color: var(--muted-alt);
+            text-align: center;
+            pointer-events: none;
+            background-color: var(--bg-light);
+            padding: 1rem;
+            opacity: 0.75;
+            transition: all 150ms;
+        }
+
+        .NotebookPanelDropzone__dropped {
+            overflow: hidden;
+            flex: 1;
+            display: flex;
+            flex-direction: column;
+        }
+
+        &--active {
```

Comment:
I don't quite follow? If I understand the SCSS spec correctly, it would result in the "--active" class being after the base one, so it would still override the base.

What's the problem that you're thinking of that can come up in this case?